node/subsystem-util/src/lib.rs

```diff
@@ -72,6 +72,12 @@ pub mod runtime;
 /// Database trait for subsystem.
 pub mod database;
 
+/// Nested message sending.
+///
+/// Useful for having mostly synchronous code, with submodules spawning short lived asynchronous
+/// tasks, sending messages back.
+pub mod nesting_sender;
```

Comment:
nit: I noticed in the generated rustdoc that everything after the first line is not included. I think we need something like this (unformatted):

```suggestion
/// Nested message sending. Useful for having mostly synchronous code, with submodules spawning short lived asynchronous
/// tasks, sending messages back.
pub mod nesting_sender;
```

Reply:
it is, but not in the overview. Fixed.